content/posts/pull-from-upstream.md

```diff
@@ -0,0 +1,116 @@
+to `propose_downstream`, but is configured directly in the dist-git repository.
+Let's now look at how to set it up and how it works.
+
+## Setup
+
+### Upstream release monitoring
+
+`pull_from_upstream` job reacts to a new bug in [Bugzilla](https://bugzilla.redhat.com/) about a new upstream version
+of a project. The bug is automatically created by
+[Upstream Release Monitoring](https://docs.fedoraproject.org/en-US/package-maintainers/Upstream_Release_Monitoring/).
+To enable the Upstream Release Monitoring:
+{{< columns >}}
+
+1. Add the upstream project (if it is not there yet)
+   to [Anitya](https://release-monitoring.org/) and configure the mapping to a Fedora package:
+   ![Project in Anitya](/images/anitya-project.png)
+   <--->
+2. Enable the monitoring in the dist-git repository ([Fedora Package Sources](https://src.fedoraproject.org)):
+
+![Monitoring in dist-git](/images/dist-git-monitoring.png)
+
+{{< /columns >}}
+
+{{< hint info >}}
+In Anitya, there are multiple backends you can configure the mapping for.
+Besides GitHub or GitLab, you can use e.g. PyPI, pagure, or
+[many others](https://release-monitoring.org/static/docs/user-guide.html#backends).
+Also, be aware that there can be a delay in retrieving the new version,
+so the update to Fedora is usually not created instantly.
+{{< /hint >}}
+
+### Packit configuration
+
+To automatically pull the upstream release as a reaction to the bug in Bugzilla, `pull_from_upstream` job
+together with the `upstream_project_url` configuration option, needs to be defined in the default branch
+(`rawhide`) of the dist-git repository in the Packit configuration file (see
+[our documentation](/docs/configuration#pull_from_upstream)). The `upstream_project_url` needs to be a URL
+pointing to a Git repository so that we can do `git` commands on it.
+
+## `pull_from_upstream` in action
+
+Let's showcase the new job in action for the latest release of
+[Packit itself](https://pypi.org/project/packitos/).
+
+As you can see in the `Setup` section, the Upstream Release Monitoring is configured:
+there is a PyPI project `packitos` in Anitya
+with configured mapping to the Fedora package `packit` and the monitoring in the
+[`packit` dist-git repository](https://src.fedoraproject.org/rpms/packit) is enabled.
+We could configure the mapping in Anitya from the GitHub project directly instead, and it would work as well. Just be aware that
+for each Fedora package, there can be a mapping only from one project.
+
+In Packit configuration file, we have configured the job and related options:
+
+```yaml
+upstream_project_url: https://github.com/packit/packit
+issue_repository: https://github.com/packit/packit
+copy_upstream_release_description: true
+
+jobs:
+  - job: pull_from_upstream
+    trigger: release
+    dist_git_branches:
+      - fedora-all
+      - epel-8
+```
+
+You can see that version `0.66.0` of Packit in PyPI was released:
```

Comment:
It might be confusing that the screenshot says `packitos`. I know the packitos->Packit mapping can be seen on the Anitia screenshot, but I'd still mention it so it's obvious that it's not a mistake
```suggestion
You can see that version `0.66.0` of Packit (packitos in PyPI) was released:
```

Reply:
I know you've already prepared the screenshots, but as Jirka has pointed out, `packit` is not the best example. I would prefer `ogr` as an example. 

And we can reuse/share some screenshots with https://packit.dev/posts/downstream-automation/ and/or https://packit.dev/docs/fedora-releases-guide/ )

Reply:
I chose Packit because there was already a release triggering `pull_from_downstream`, for `ogr` there was not yet such a release. 

I thought it could be beneficial to show that people can also use PyPI as a backend, but we can also show `specfile` if you think this is confusing, WDYT?

Reply:
If you mention that naming thing as Jirka has suggested, I would not spend the time on doing the screenshots again.